ONSdigital/dp-developer-site
# Feature/tour of ons api

### What

- Added Webpack 4 configuration with Babel and eslint for building js files.

*Implemented ONS API tour functionality:*

- Pattern and styling updated
- Show/hide Javascript example button
- Try it out button
- Results - JSON, formatted table, line chart (using HighCharts)
- Tabs component to toggle between results: JSON/Content 

### How to review

- Pull repo.
- Running `make watch-serve` should build the site in its entirety. You can also run `npm run build:js` and `npm run build:css` for JS and CSS files respectively.
- Navigate to the tour by clicking on, *Take a tour of the API* from the nav.
- Run through the tour, checking that the copy, example and results correspond to the correct endpoints.
- Relevant files can be found in `/static/tour`, `/js/tour`, `/scss/elements/_table.scss` and `/scss/elements/_tour.scss`

### Who can review

Anyone but me.
